src/components/ui/tiptap-ui-primitive/popover/popover.tsx

```diff
@@ -264,6 +264,8 @@ interface PopoverContentProps extends React.HTMLProps<HTMLDivElement> {
   portal?: boolean
   portalProps?: Omit<React.ComponentProps<typeof FloatingPortal>, "children">
   asChild?: boolean
+  modal?: boolean
+  initialFocus?: number | React.MutableRefObject<HTMLElement | null>
```

Comment:
I'm not sure I understand why `React.MutableRefObject<HTMLElement | null>` is used here or what it's for. If this is idiomatic maybe we can document why it's used? Otherwise perhaps we can use something more primitive?

Reply:
Great suggestion.